src/test/groovy/com/intuit/graphql/orchestrator/integration/QueryInterfaceSpec.groovy

```diff
@@ -0,0 +1,110 @@
+package com.intuit.graphql.orchestrator.integration
+
+import com.intuit.graphql.orchestrator.*
+import graphql.ExecutionInput
+import graphql.execution.AsyncExecutionStrategy
+import helpers.BaseIntegrationTestSpecification
+
+class QueryInterfaceSpec extends BaseIntegrationTestSpecification {
```

Comment:
Looks good but missing the validation of execution input for the downstream service.